tf_agents/utils/batched_observer_unbatching_example.py

```diff
@@ -0,0 +1,66 @@
+# coding=utf-8
```

Comment:
This file only serves the purpose of testing. You can put it in the batched_observer_unbatching_example_test.py.

Reply:
Done.